tools/ratbag-command.c

```diff
@@ -1435,6 +1455,292 @@ static const struct ratbag_cmd cmd_button = {
 };
 
 static int
+ratbag_cmd_led_set_bright(const struct ratbag_cmd *cmd,
+    struct ratbag *ratbag,
+    struct ratbag_cmd_options *options,
+    int argc, char **argv) {
+	struct ratbag_device *device;
+	struct ratbag_led *led;
+	char* str, *endptr;
+	int bright, b;
+	int rc;
+
+	if (argc < 1)
+		return ERR_USAGE;
+
+	led = options->led;
+	str = argv[0];
+	bright = strtoul(str, &endptr, 10);
```

Comment:
If there's no real error checking, we should just used atoi, or copy the safe_atoi wrapper from libinput (which I wrote, so feel free to copy it :)

(that also applies to a bunch of other strtol already there)